### ISSUE TITLE
Can't use tabs for indentation

### ISSUE THREAD

**Issue by satsen** [casid/jte-intellij#21]
Even though I have configured indentation to be tabs for JavaTemplateEngine, it ignores this and writes 4 spaces when I press tab.

Toggling indents detection doesn't change anything.

IntelliJ version: 2022.1.3
Plugin version: 2.0.7

**Comment by casid** [comment]
Welcome, @satsen and thanks for reporting.

Did you change the settings here? This is working for me (IntelliJ 2021.3):
![image](https://user-images.githubusercontent.com/4077354/179891353-91826570-83ec-4211-a5b8-00949641cbf7.png)

Another pitfall might be that there are separate settings for jte files and kte files (in case you might use Kotlin).

**Comment by satsen** [comment]
Thanks, that works, but from the bottom bar under the editor changing it has no effect. It seems to configure another file type called "JavaTemplateEngine" while in the settings there is "jte".

**Comment by satsen** [comment]
@casid

**Comment by casid** [comment]
Hey @satsen, sorry for the late reply.

It took some reverse engineering to find out what the problem was. `JavaTemplateEngine` is the internal ID of the jte language plugin. The display name of the code config option pane is `jte`, though. Apparently, that is not just the display name, but also the way IntelliJ finds out what code config option pane should be opened in `com.intellij.psi.codeStyle.statusbar.CodeStyleStatusBarWidgetFactory#createDefaultIndentConfigureAction`:

```java
@NotNull
public static DumbAwareAction createDefaultIndentConfigureAction(@NotNull PsiFile psiFile) {
  String langName = getLangName(psiFile);
  return DumbAwareAction.create(
    ApplicationBundle.message("code.style.widget.configure.indents", langName),
    event -> {
      Configurable configurable = findCodeStyleConfigurableId(psiFile.getProject(), langName);
      if (configurable instanceof CodeStyleConfigurableWrapper) {
        ShowSettingsUtil.getInstance().editConfigurable(
          event.getProject(), configurable,
          () -> {
            CodeStyleConfigurableWrapper configurableWrapper = (CodeStyleConfigurableWrapper)configurable;
            configurableWrapper.setSchemesPanelVisible(false);
            configurableWrapper.selectTab(ApplicationBundle.message("title.tabs.and.indents"));
          });
      }
    }
  );
}
```

I've adjusted both display names to match each other, now the correct options are opened :-)

I just published version `2.0.8`, it should be available for download after passing JetBrains review (usually takes about two business days).

**Comment by satsen** [comment]
@casid Thank you for fixing it and sharing detailed information about the cause! I just received the update configuring from the bottom bar works as expected now.
